src/ScoreSystem.js

```diff
@@ -0,0 +1,27 @@
+
+export class ScoreSystem {
+	constructor() {
```

Comment:
You can omit the constructor if
* it is empty
* it delegates all arguments to the super-constructor